lib/ui/views/groups/group_details_view.dart

```diff
@@ -385,3 +397,32 @@ class _GroupDetailsViewState extends State<GroupDetailsView> {
     );
   }
 }
+
+class AddButton extends StatefulWidget {
```

Comment:
Code repetition in Add Button, can be moved to a separate widget. Also try naming it in a better way for the meaning to be understood by name itself.